internal/ecsservicediscovery/servicediscovery.go

```diff
@@ -31,8 +33,17 @@ func (sd *ServiceDiscovery) init() {
 	}
 	configProvider := credentialConfig.Credentials()
 	sd.svcEcs = ecs.New(configProvider, aws.NewConfig().WithRegion(sd.Config.TargetClusterRegion).WithMaxRetries(AwsSdkLevelRetryCount))
+	if err := sd.Configurer.Configure(awsmiddleware.SDKv1(&sd.svcEcs.Handlers)); err != nil {
+		log.Printf("ERROR: Failed to configure ECS client: %v", err)
```

Comment:
Should we remove this? I think since it just for the init function it is reasonable to have.